components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/internal/CertValidationDataHolder.java

```diff
@@ -20,6 +20,7 @@
 
 import org.wso2.carbon.identity.certificate.management.service.CertificateManagementService;
 import org.wso2.carbon.identity.configuration.mgt.core.ConfigurationManager;
+import org.wso2.carbon.identity.x509Certificate.validation.service.CertificateValidationManagementService;
```

Comment:
what is the use of this import

Reply:
fixed with https://github.com/wso2-extensions/identity-x509-commons/pull/57/commits/2f9c4cc0ce01a4d48b026d0659ff2194488b9c8a